### ISSUE TITLE
AttributeError: 'Command' object has no attribute 'LANGUAGES'

### ISSUE THREAD

**Issue by matiaslina** [django-extensions/django-extensions#1383]
The command `show_urls` raises an AttributeError running without any flags on version 2.1.7.

```
% ./manage show_urls
Traceback (most recent call last):
  File "/home/matias/projects/wop/wop/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/matias/.virtualenvs/wop/lib/python3.6/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/home/matias/.virtualenvs/wop/lib/python3.6/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/matias/.virtualenvs/wop/lib/python3.6/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/matias/.virtualenvs/wop/lib/python3.6/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/home/matias/.virtualenvs/wop/lib/python3.6/site-packages/django_extensions/management/utils.py", line 62, in inner
    ret = func(self, *args, **kwargs)
  File "/home/matias/.virtualenvs/wop/lib/python3.6/site-packages/django_extensions/management/commands/show_urls.py", line 121, in handle
    view_functions = self.extract_views_from_urlpatterns(urlconf.urlpatterns)
  File "/home/matias/.virtualenvs/wop/lib/python3.6/site-packages/django_extensions/management/commands/show_urls.py", line 227, in extract_views_from_urlpatterns
    for language in self.LANGUAGES:
AttributeError: 'Command' object has no attribute 'LANGUAGES'
```

The bug happen on https://github.com/django-extensions/django-extensions/commit/b58b3320c2ffc2fad964e0c9dfa24363a2fde645 removing the initialization of `self.LANGUAGES` at line 93

As a workaround, version 2.1.6 works and `./manage.py show_urls --language=en` in version 2.1.7 also works.

**Comment by trbs** [comment]
@matiaslina could you please confirm 8b53b18 fixes this for you ? if it's confirmed to be working I will make a new release as well :-)

**Comment by matiaslina** [comment]
I did a diff between 2.1.6 and that particular commit with the output of `./manage.py show_urls` and got the same output.

So it seems that https://github.com/django-extensions/django-extensions/commit/8b53b186555882170beef8613b7081e27cf27c2f fixes the problem.

Thanks @trbs ! :)

**Comment by trbs** [comment]
awesome, released 2.1.9 with the fix :-)
